claim-check/code-samples/sample-2/client-consumer/EventHubsConsumer.cs

```diff
@@ -76,48 +35,79 @@ public void Configure()
 
             string storageConnectionString = ConfigurationManager.AppSettings["StorageConnectionString"];
             string eventhubConnectionString = ConfigurationManager.AppSettings["EventHubConnectionString"];
-
             downloadDestination = ConfigurationManager.AppSettings["DownloadDestination"];
-
+            string blobContainerName = ConfigurationManager.AppSettings["BlobContainerName"];
             Console.WriteLine("Connecting to Storage account...");
-
-            storageAccount = CloudStorageAccount.Parse(storageConnectionString);
-
+            BlobContainerClient blobContainerClient = new BlobContainerClient(storageConnectionString, blobContainerName);
             Console.WriteLine("Connecting to EventHub...");
-            
-            client = EventHubClient.CreateFromConnectionString(eventhubConnectionString);
+            processor = new EventProcessorClient(blobContainerClient, EventHubConsumerClient.DefaultConsumerGroupName, eventhubConnectionString);
         }
 
         public async Task ProcessMessages(CancellationToken cancellationToken)
         {
             Console.WriteLine("The application will now start to listen for incoming message.");
+            int eventIndex = 0;
 
-            var runtimeInfo = await client.GetRuntimeInformationAsync();
-            Console.WriteLine("Creating receiver handlers...");
-            var utcNow = DateTime.UtcNow;
-            var receivers = runtimeInfo.PartitionIds
-                .Select(pid => {
-                    var receiver = client.CreateReceiver("$Default", pid, EventPosition.FromEnqueuedTime(utcNow));
-                    Console.WriteLine("Created receiver for partition '{0}'.", pid);
-                    receiver.SetReceiveHandler(new PartitionReceiver(storageAccount, downloadDestination));
-                    return receiver;
-                })
-                .ToList();
+            Task processEventHandlerAsync(ProcessEventArgs eventArgs)
+            {
+                if (eventArgs.CancellationToken.IsCancellationRequested)
+                {
+                    return Task.CompletedTask;
+                }
+                try
+                {
+                    ++eventIndex;
+                    Console.WriteLine($"Event Received: { Encoding.UTF8.GetString(eventArgs.Data.Body.ToArray()) }");
+                    string body = Encoding.UTF8.GetString(eventArgs.Data.Body.ToArray());
```

Comment:
Don't duplicate the GetString logic.  Put in variable before you do a console writeline.